app/helpers/katello/subscription_mailer_helper.rb

```diff
@@ -0,0 +1,25 @@
+module Katello
+  module SubscriptionMailerHelper
+    include Rails.application.routes.url_helpers
+
+    def report_url
+      report_id = ReportTemplate.find_by(name: "Subscription - Entitlement Report").id
```

Comment:
These queries, I think, would be better off taking advantage of the helper and create methods, such as this:

```
def report_template
  @report_template ||= ReportTemplate.find_by_name("Subscription - Entitlement Report")
end
```

Idea here is that we can use this in multiple places, and memoize the value so the query will be ran only once. we can also grab more data than just the `id` if we need to for some reason later on, with fewer changes to the code.